camel-example-spring-boot-load-balancer-eip/src/main/java/sample/camel/LoadBalancerEIPRouter.java

```diff
@@ -0,0 +1,120 @@
+package sample.camel;
```

Comment:
License is missing and please use the package org.apache.camel.springboot.example